onnx_caffe2/backend.py

```diff
@@ -197,6 +200,33 @@ def _create_constant(cls, n):
         assert len(n.outputs) == 1
         return cls._create_tensor_filling_op(n.attrs["value"], n.outputs[0])
 
+    @classmethod
+    def _create_gemm(cls, n):
+        (A, B, C) = n.inputs
+        (Y,) = n.outputs
+        alpha = n.attrs.get('alpha', 1)
+        beta = n.attrs.get('beta', 1)
+
+        ops = []
+        if alpha != 1:
+            scaled_A = dummy_name()
+            ops.append(core.CreateOperator('Scale', [A], [scaled_A], scale=alpha))
+            A = scaled_A
+        if beta != 1:
+            scaled_C = dummy_name()
+            ops.append(core.CreateOperator('Scale', [C], [scaled_C], scale=beta))
+            C = scaled_C
+
+        AB = dummy_name()
+        ops.append(core.CreateOperator('MatMul',
+                                       [A, B],
+                                       [AB],
+                                       trans_a=n.attrs.get('transA', False),
+                                       trans_b=n.attrs.get('transB', False)))
+        ops.append(core.CreateOperator('Add', [AB, C], [Y]))
```

Comment:
Does this work with both settings of `broadcast`? If so, worth a comment saying so.

Reply:
Actually, caffe2's Add operator also needs to specify the "broadcast" argument.